src/Error/ErrorHandlerInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Digia\GraphQL\Error;
+
+interface ErrorHandlerInterface
+{
+    /**
+     * @param ExecutionException $error
+     */
+    public function handleError(ExecutionException $error);
```

Comment:
Here it's called `$error`, in the implementation it's called `$exception`

Reply:
Good catch, I'll fix it.